File: imagepicker/src/main/java/com/nguyenhoanglam/imagepicker/ui/camera/CameraView.java
package com.nguyenhoanglam.imagepicker.ui.camera;

import com.nguyenhoanglam.imagepicker.model.Image;
import com.nguyenhoanglam.imagepicker.ui.common.MvpView;

import java.util.List;

/**
 * Created by hoanglam on 8/22/17.
 */

public interface CameraView extends MvpView {

    void finishPickImages(List<Image> images);
}

File: imagepicker/src/main/java/com/nguyenhoanglam/imagepicker/ui/camera/CameraView.kt
package com.nguyenhoanglam.imagepicker.ui.camera

import com.nguyenhoanglam.imagepicker.model.Image
import com.nguyenhoanglam.imagepicker.ui.common.MvpView

/**
 * Created by hoanglam on 8/22/17.
 */
interface CameraView : MvpView {
    fun finishPickImages(images: List<Image>)
}